docs/scripting/language/Style.md

```diff
@@ -221,22 +221,29 @@ Enumerators, if named, must be prefixed with `E_` and use
 `Index tag mismatch` warning, in which case an `e_` prefix may be used.
 `SCREAMING_SNAKE_CASE` must still be used even with the lower case `e_` prefix.
 
-Enumerator fields must also be `SCREAMING_SNAKE_CASE` and use the enumerator
-name as a prefix.
-
 ```pawn
-static enum E_PLAYER {
-    E_PLAYER_CASH,
-    Float:E_PLAYER_HEALTH,
+static enum E_PLAYER_DATA {
+    e_PlayerCash,
+    Float:e_PlayerHealth,
 }
 ```
 
 To avoid index tag warnings in some cases:
```

Comment:
That's not entirely true.  If you're using an enum as an index, you are best off using `E` not `e`.  The difference is to do with strong/weak tags.

Reply:
This comment was not added by me it was there earlier, and I know about the difference in tags I thought about adding that but left it for another PR